scipy/stats/_multivariate.py

```diff
@@ -309,7 +313,11 @@ class multivariate_normal_gen(multi_rv_generic):
     -----
     %(_mvn_doc_callparams_note)s
 
-    The covariance matrix `cov` must be a symmetric positive semidefinite
+    The covariance matrix `cov` may be an instance of a subclass of
+    `Covariance`, e.g. `scipy.stats.CovViaPrecision`. If so, `allow_singular`
```

Comment:
```
/home/circleci/repo/doc/docstring of scipy.stats.multivariate_normal:44: WARNING: py:obj reference target not found: scipy.stats.CovViaPrecision
```
Not sure what the best way to fix this is. I don't really know the rules by which Sphinx looks things up. This is a public class and it's in the present subpackage, `stats` - is `CovViaPrecision` enough?

Reply:
The documentation for `CovViaPrecision` wasn't being generated because it was missing
```
.. autosummary::
   :toctree: generated/
```
in `__init__.py`. Hopefully this is fixed by the latest commit.